src/main/java/com/exactpro/epfast/decoder/presencemap/DecodePresenceMap.java

```diff
@@ -0,0 +1,80 @@
+package com.exactpro.epfast.decoder.presencemap;
+
+import com.exactpro.epfast.decoder.IDecodeContext;
+import io.netty.buffer.ByteBuf;
+
+import java.util.BitSet;
+
+public class DecodePresenceMap implements IDecodeContext {
```

Comment:
I was wrong that we need `getValue(index)` on `DecodePresenceMap`. We need separate `PresenceMap` class and `DecodePresenceMap.getValue()` should return it. I'm pretty sure multiple presence map can be active during decoding